minifi-bootstrap/src/main/java/org/apache/nifi/minifi/bootstrap/util/ConfigTransformer.java

```diff
@@ -428,13 +436,26 @@ private static void addProcessGroup(final Element parentElement, Map<String, Obj
             addPosition(element);
             addTextElement(element, "comment", getValueString(flowControllerProperties, COMMENT_KEY));
 
-            Map<String, Object> processorConfig = (Map<String, Object>) topLevelYaml.get(PROCESSOR_CONFIG_KEY);
-            addProcessor(element, processorConfig);
+            Collection<Object> processors = (Collection<Object>) topLevelYaml.get(PROCESSORS_KEY);
+            if (processors != null) {
+                for (Object processorConfig : processors) {
+                    addProcessor(element, (Map<String, Object>) processorConfig);
+                }
+            }
 
-            Map<String, Object> remoteProcessingGroup = (Map<String, Object>) topLevelYaml.get(REMOTE_PROCESSING_GROUP_KEY);
-            addRemoteProcessGroup(element, remoteProcessingGroup);
+            Map<String, Object> exfilComponent = (Map<String, Object>) topLevelYaml.get(EXFIL_COMPONENT_KEY);
+            if ("Site-To-Site".equals(getValueString(exfilComponent, TYPE_KEY))) {
```

Comment:
minor:  would be nice to make this a constant as it is used in two different places for checks